index.html

```diff
@@ -363,7 +370,7 @@ <h4 class="ibm-padding-top-0">Roles now filled.</h4>
                                 <div class="ibm-fluid ibm-padding-top-3 ibm-padding-bottom-2">
 
                                     <div class="ibm-col-12-12 cio-info-grid">
-                                        <div class="cio-info-grid__hero-image cio-image" style="background-image: url(assets/images/team.jpg)">
+                                        <div class="cio-info-grid__hero-image cio-image" style="background-image: url(assets/images/work-space-1.jpg)">
```

Comment:
Could we remove the old images that aren't being used anymore please?

Reply:
Not sure I understand - 'team' is the old image and 'work-space-1' is the new image Ryan requested.  Let's chat when you're back at your desk.

Reply:
Thanks for explaining - have deleted.